### PR TITLE
Contribution guidelines

### PR DESCRIPTION
**Pull request by ValentinaHutter**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+## Adding new processes
+
+If you only want to update implementation details in a process, the specification should remain as it is and you do not need to update the submodule.
+
+If you want to add a new process or update the parameters of the process, you will also need to interact with the submodule.
+
+The specifications come from a fork of the official openeo-processes: https://github.com/eodcgmbh/openeo-processes
+
+To add a new process:
+- add the specification to https://github.com/eodcgmbh/openeo-processes
```
Comment (clausmichele) [inline .github/CONTRIBUTING.md:107]: Maybe specify: create a fork, add your process to the root folder, moving it from the missing-processes folder or creating a new process definition.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+- update the submodule in openeo-processes-dask
+- add the implementation in openeo-processes-dask
+- cover the new implementation in the tests
+- update the dependencies, if you need to introduce a new package. `poetry add ...`.
+
+The specifications can then be used to create a process registry, e.g. https://github.com/Open-EO/openeo-pg-parser-networkx/blob/main/examples/01_minibackend_demo.ipynb
```
Comment (clausmichele) [inline .github/CONTRIBUTING.md:113]: I usually test new implementations using the client side processing example instead, which uses directly the openeo python client https://open-eo.github.io/openeo-python-client/cookbook/localprocessing.html
The example in the notebook might be quite complex to understand for a new user.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+```
+from openeo_processes_dask.specs import load_collection as load_collection_spec
+process_registry["load_collection"] = Process(spec=load_collection_spec, implementation=load_collection)
+```
+
+## Prior to submitting a PR - a checklist
```
Comment (clausmichele) [inline .github/CONTRIBUTING.md:119]: Add the pre-commit part?

Comment (ValentinaHutter) [reply .github/CONTRIBUTING.md:131]: resolving this as the pre-commit part is the third point below.

**Pull request merged**